skrub/tests/test_joiner.py

```diff
@@ -2,25 +2,15 @@
 import pandas as pd
 import pytest
 from numpy.testing import assert_array_equal
-from pandas.testing import assert_frame_equal
 
 from skrub import Joiner
-from skrub._dataframe._polars import POLARS_SETUP
+from skrub._dataframe import _common as ns
+from skrub._dataframe._test_utils import assert_frame_equal
 
-MODULES = [pd]
-ASSERT_TUPLES = [(pd, assert_frame_equal)]
 
-if POLARS_SETUP:
-    import polars as pl
-    from polars.testing import assert_frame_equal as assert_frame_equal_pl
-
-    MODULES.append(pl)
-    ASSERT_TUPLES.append((pl, assert_frame_equal_pl))
-
-
-@pytest.mark.parametrize("px", MODULES)
-def test_joiner(px):
-    main_table = px.DataFrame(
+# TODO: change this name
```

Comment:
could you either rename it in this PR or remove the comment or make it more explicit to explain why it should be changed and why it's not done yet?

Reply:
Yes I wanted to change it because
- having a test named `test_joiner` in the `test_joiner.py` file isn't very informative
- I wasn't able to run this test only using `pytest -k 'test_joiner'` because it shares name with the file

It's possible we want to split it into something like `test_fit_transform`, `test_wrong_main_key`, `test_wrong_aux_key`